internal/bminventory/inventory.go

```diff
@@ -1965,10 +1951,27 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 		}
 	}
 
+	if params.ClusterUpdateParams.MachineNetworkCidr != nil && common.IsSingleNodeCluster(cluster) {
+		err = network.VerifyMachineCIDR(swag.StringValue(params.ClusterUpdateParams.MachineNetworkCidr), cluster.Hosts, log)
+		if err != nil {
+			return err
+		}
+		err = setCommonUserNetworkManagedParams(params.ClusterUpdateParams, cluster, updates, log)
+		if err != nil {
+			return err
+		}
+		machineCidr = swag.StringValue(params.ClusterUpdateParams.MachineNetworkCidr)
```

Comment:
Move to line 1955, and use it when calling `network.VerifyMachineCIDR`

Reply:
nope, it need to have machineCidr var updated, it will be use in further validations

Reply:
missed the line number, will fix